README.md

```diff
@@ -1,9 +1,55 @@
 # Gruntwork Installer
 
-At [Gruntwork](http://www.gruntwork.io/), we've developed a number of scripts and binaries, most of them in private
+`gruntwork-install` is a bash script you run to easily download and install "Script Modules" written by Gruntwork. 
+
+### Script Modules
+
+A Script Module is a package of one or more bash scripts and/or binaries maintained by Gruntwork that are meant to be 
+reusable in many different contexts. For example, we have Script Modules for installing a CloudWatch Logs agent, optimizing
+syslog settings, setting up automatic security updates, and more. 
+
+Our Script Modules are contained in the `modules/` folder of different GitHub repos since they often come with the 
+different [Infrastructure Packages](https://blog.gruntwork.io/gruntwork-infrastructure-packages-7434dc77d0b1#.6bwor6wxc) 
+we sell.
+
+#### Common Script Modules
+
+Some Script Modules are so common that we've made them freely available in the [modules/](modules) folder of this repo.
+
+### Motivation
+At [Gruntwork](http://www.gruntwork.io/), we've developed a number of scripts and binaries, most of them in private GitHub
 repos, that perform common infrastructure tasks such as setting up continuous integration, monitoring, log aggregation,
-and SSH access. This repo provides a script called `gruntwork-install` that makes it as easy to install Gruntwork
-scripts and binaries as using apt-get, brew, or yum.
+and SSH access. Being able to use these "modules" of code typically involves many steps: you download the files 
+(possibly from a private GitHub repo), change their permissions, and run them with the parameters that make sense for 
+your environment.
+
+That basically means lots of custom `bash` code copied differently across multiple software teams. Worse, if we 
+want to update a binary or script to add a new parameter, each team has to modify their own custom code, which can be 
+painful. 
+
+We believe we can do better by writing our scripts and binaries in a standardized way, and including a minimal tool that 
+streamlines the process of downloading and installing them. Indeed, our goal is to make installing Gruntwork Script 
+Modules as easy as installing a typical package using `apt-get`, `yum`, `npm`, or similar tools. And since we give you 
+100% of the source code, we want it to be clear exactly what happens when you install a Gruntwork Script Module.  
```

Comment:
One important part of the motivation: we need an install that works across a variety of different (*nix) operating systems, so we can't use apt-get or yum directly.